File: inership_project/Task 1/num_guess.py
import random

def guess_number():
    secret_number = random.randint(1, 100)

    print("Guess the number between 1 and 100: ")

    guess = None
    while guess != secret_number:
        guess = int(input("Enter your guess: "))

        if guess < secret_number:
            print("Too low! Try again.")
        elif guess > secret_number:
            print("Too high! Try again.")
        else:
            print("Congratulations! You've guessed the correct number!")

guess_number()

File: Num_guess.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;

int main() {
    int secretNumber, guess;
    srand(time(0)); // Seed for random number generator
    secretNumber = rand() % 100 + 1; // Generates a number between 1 and 100

    cout << "Guess the number between 1 and 100: ";
    
    do {
        cin >> guess;

        if (guess < secretNumber) {
            cout << "Too low! Try again: ";
        } else if (guess > secretNumber) {
            cout << "Too high! Try again: ";
        } else {
            cout << "Congratulations! You've guessed the correct number!" << endl;
        }

    } while (guess != secretNumber);

    return 0;
}
